src/tarantool-client/Converters/AuthenticationPacketConverter.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+
+using iproto.Data;
+using iproto.Data.Packets;
+
+using TarantoolDnx.MsgPack;
+
+namespace tarantool_client.Converters
+{
+    public class AuthenticationPacketConverter : IMsgPackConverter<AuthenticationPacket>
+    {
+        public void Write(AuthenticationPacket value, IBytesWriter writer, MsgPackContext context)
+        {
+            if (value == null)
+            {
+                context.NullConverter.Write(null, writer, context);
+            }
```

Comment:
Return here will render else redundant. This will result in less nested, more readable code